apps/sentry/src/env.ts

```diff
@@ -16,6 +47,9 @@ export const env = z
     DATABASE_PROXY_PORT: z.coerce.number().int().positive(),
     VERCEL_PREFERRED_REGION: z.string().min(1),
     VERCEL_ENV: z.string().min(1).optional(),
-    ENCRYPTION_KEY: z.string().length(64),
```

Comment:
include the ENCRYPTION_KEY too as it is the addition by elba team.

Reply:
Done